docs/core/testing/unit-testing-mstest-configure.md

```diff
@@ -100,3 +100,97 @@ Each element of the file is optional because it has a default value.
 
 </RunSettings>
 ```
+
+## testconfig.json
+
+Starting with MSTest 3.7, when running your tests through MSTest runner (a.k.a Microsoft.Testing.Platform), you can use a `testconfig.json` file to configure the behavior of the test runner. The `testconfig.json` file is a JSON file that contains the configuration settings for the test runner. The file is used to configure the test runner and the test execution environment. For more information, refer to [Microsoft.Testing.Platform testconfig.json documentation](unit-testing-platform-config.md#testconfigjson).
+
+### MSTest element
+
+MSTest settings are grouped by functionality that are described in the sections below.
+
+| Entry | Default | Description |
```

Comment:
Is Execution header missing here?

Reply:
Nope it's some error in what was done and slipped through review. I am planning on fixing code and doing a follow up on doc but at least I was sure not to miss anything.

Reply:
where is the error? The settings you list here are shown in the "execution" section in the example json file.

Reply:
It is me who is confusing stuff here. I somehow thought these settings are the ones that are in the execution section, and they are not.